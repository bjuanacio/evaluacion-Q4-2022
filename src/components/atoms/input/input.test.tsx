import { render } from "@testing-library/react";
import Input from "./input";

describe("button", () => {
  test("should have the correct style", () => {
    const { container } = render(<Input />);
    expect(container).toMatchSnapshot();
  });
});
