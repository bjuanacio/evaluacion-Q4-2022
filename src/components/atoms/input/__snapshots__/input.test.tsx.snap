// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button should have the correct style 1`] = `
<div>
  <input
    class="input"
  />
</div>
`;
